overlord/state/state.go

```diff
@@ -92,12 +92,13 @@ type State struct {
 	// for registering runtime callbacks
 	lastHandlerId int
 
-	backend  Backend
-	data     customData
-	changes  map[string]*Change
-	tasks    map[string]*Task
-	warnings map[string]*Warning
-	notices  map[noticeKey]*Notice
+	backend        Backend
+	data           customData
+	changes        map[string]*Change
+	tasks          map[string]*Task
+	warnings       map[string]*Warning
+	notices        map[noticeKey]*Notice
+	noticeLastDate time.Time
```

Comment:
I think this should be named `noticeLastOccurred` as occurred / last occurred is what that field is called. Similar for the JSON field.